integrationTests/testProcessorNodeWithMultisigner.go

```diff
@@ -249,8 +249,8 @@ func createInitialNodes(validatorsMap map[uint32][]sharding.Validator, waitingMa
 	for _, pks := range waitingMap {
 		for _, validator := range pks {
 			n := &sharding.InitialNode{
-				PubKey:   core.ToHex(validator.PubKey()),
-				Address:  core.ToHex(validator.Address()),
+				PubKey:   TestValidatorPubkeyConverter.Encode(validator.PubKey()),
+				Address:  TestAddressPubkeyConverter.Encode(validator.Address()),
```

Comment:
shoudn't this be still be encode(validator.PubKey)?

Reply:
You mean using hex directly? Stated above

Reply:
carified